azext_edge/edge/providers/support/connectors.py

```diff
@@ -34,7 +31,7 @@
 CONNECTORS_DIRECTORY_PATH = "connectors"
 
 # TODO: once this label is stabled, we can remove the other labels
-OPCUA_NAME_LABEL = NAME_LABEL_FORMAT.format(label=OPCUA_API_V1.label)
+OPCUA_NAME_LABEL = NAME_LABEL_FORMAT.format(label="microsoft-iotoperations-opcuabroker")
```

Comment:
Is this the only place the opcua label is used?

Reply:
yes, check is importing the label from here